### ISSUE TITLE
WeChat Mini Programs cannot connect in real time

### ISSUE THREAD

**Issue by jericho2018** [feathersjs/feathers#1802]
WeChat Mini Programs real-time connection, must use wws:// service, how to use feathers to build? The official website only mentions the http method. Hope that the great feathers can consider the use of Mini Program, there is a huge market!

**Comment by jericho2018** [comment]
and the feathers-client cannot take effect in Mini Program

**Comment by daffl** [comment]
How to set up HTTPS is [documented here](https://docs.feathersjs.com/api/express.html#https).

**Comment by jericho2018** [comment]
Thank you very much. I have solved this https problem. The new problem encountered at present is that WeChat Mini Programs natively only supports websocket connections, but not socket.io, and then I modified the configuration code on the server side:

app.js:
app.configure(socketio({ transports: ['websocket'] }）

but still unable to connect, can i let feathers use websocket to connect?

**Comment by jericho2018** [comment]
> How to set up HTTPS is [documented here](https://docs.feathersjs.com/api/express.html#https). Another common way is to use the load balancer (e.g. NginX) to provide SSL.

Thank you very much. I have solved this https problem. The new problem encountered at present is that WeChat Mini Programs natively only supports websocket connections, but not socket.io, and then I modified the configuration code on the server side:

app.js:
app.configure(socketio({ transports: ['websocket'] }）

but still unable to connect, can i let feathers use websocket to connect?
